microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -14,7 +14,7 @@ As a Microsoft Edge extension developer, you can make your new or existing Micro
 
 ![The sidebar for a Microsoft Edge extension](./sidebar-images/sidebar-screenshot.png)
 
-Note: The sidebar extensions feature is being rolled out to a growing user base for all channels of Microsoft Edge.<!-- todo: after rollout is complete, delete sentence -->
+The sidebar extensions feature is being rolled out to a growing user base for all channels of Microsoft Edge.<!-- todo: after rollout is complete, delete sentence -->
```

Comment:
feature is being rolled out

Reply:
i deleted "The sidebar extensions feature is being rolled out to a growing user base for all channels of Microsoft Edge."
@bhuvanapriyap @captainbrosset